validator/xsd/schedule.xml.xsd

```diff
@@ -242,4 +242,17 @@
     </xs:simpleContent>
   </xs:complexType>
 
+  <xs:simpleType name="optoutEnum">
+    <xs:restriction base="xs:string">
+      <!-- Recording optout status 'false' is the normal case: We enable the live stream and create a recording, which we publish without and feedback from the speaker -->
+      <xs:enumeration value="false"/>
+      <!-- Recording optout status 'true' means: The speaker does not want a recording of any kind. We will remove all personnel, and maybe even switch off all cameras -->
+      <xs:enumeration value="true"/>
+      <!-- Recording optout status 'maybe': We are allowed to create a recording / enable the live stream; but the speaker can decide afterwards if we can publish it  -->
+      <xs:enumeration value="maybe"/>
```

Comment:
How about `undecided` instead of `maybe`?

Reply:
```suggestion
      <xs:enumeration value="undecided"/>
```